src/tools/miri/tests/fail/breakpoint.rs

```diff
@@ -1,4 +1,5 @@
 #![feature(core_intrinsics)]
+#![allow(unused_unsafe)]
```

Comment:
This is odd, the test should not be executed with old rustc.

Can you add a FIXME so we remember to remove this at some point?

Reply:
@RalfJung I didn't know whether miri needed to run with both versions or not. If miri is only supposed to run with the newly compiled compiler, then I'll just delete the unsafe block.